front/app/components/admin/SeatBasedBilling/AddModeratorsModal/AddModeratorsModal.test.tsx

```diff
@@ -6,6 +6,16 @@ import { render, screen } from 'utils/testUtils/rtl';
 
 import AddModeratorsModal from '.';
 
+jest.mock('react-dom', () => ({
```

Comment:
This was needed in all tests of components that use Modals. Found it in FullscreenPreview.test.tsx. Maybe good to mention in the FE CDM.

Reply:
Instead of adding it for each test, could you please mock it for our entire test suite by adding it to `front/internals/jest/setup.js`

Reply:
I considered that, but then it's only needed in 6 of the almost 400 tests at the moment.